src/pages/AddItem.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { withFirestore } from 'react-firestore';
+
+const AddItem = ({ firestore }) => {
+  const [name, setName] = useState('');
+
+  // Send the new item to Firebase
```

Comment:
Are these comments just to bring participants up to speed?

Reply:
Yeah. In the past, since many participants are newer to react, we've been really generous with comments so that when they jump in to work on the code, they get a better idea of what each piece does.